patterns/byod.md

```diff
@@ -0,0 +1,579 @@
+| 1088    | Personnel report the potential compromise of mobile devices, media or credentials to their organisation as soon as possible, especially if they:<br />- provide credentials, decrypt devices or have devices taken out of sight by foreign government officials<br />- have devices or media stolen that are later returned <br />- lose devices or media that are later found<br />- observe unusual behaviour of devices. | Agency to address                                            | Not applicable |
+| 1300    | Upon returning from travelling overseas with mobile devices, personnel take the following actions:<br />- sanitise and reset devices, including all media used with them<br />- decommission any physical credentials that left their possession during their travel<br />- report if significant doubt exists as to the integrity of any devices following their travel. | Agency to address                                            | Not applicable |
+| 1556    | If returning from travelling overseas with mobile devices to high/extreme risk countries, personnel take the following additional actions:<br />- reset user credentials used with devices, including those used for remote access to their organisation’s systems<br />- monitor accounts for any indicators of compromise, such as failed login attempts. | Agency to address                                            | Not applicable |
+
+### Mobile device access (BYOD Option 1)
+
+The Mobile device access option will leverage:
+
+* Azure Active Directory Multi-Factor Authentication (MFA) and Conditional Access policies
+
+* Defender for Cloud Apps to prevent data leakage and spills
+
+* Intune - Application Protection Policies.
```

Comment:
Could you please remove the extra spaces between the list items as these will convert to HTML open/close ULs